logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManager.kt

```diff
@@ -184,9 +194,19 @@ actual class CallManager(
                 userId = message.senderUserId.asString(),
                 clientId = message.senderClientId.value
             )
-            kaliumLogger.d("onCallingMessageReceived -> Passed through")
+            kaliumLogger.d("$TAG - onCallingMessageReceived")
         }
 
+    actual suspend fun answerCall(conversationId: ConversationId) = withCalling {
+        kaliumLogger.d("$TAG -> answerCall")
+        calling.wcall_answer(
+            inst = deferredHandle.await(),
+            conversationId = conversationId.asString(),
+            callType = CallType.CALL_TYPE_NORMAL,
```

Comment:
What does `calLType` do here?

Could we document this?

Reply:
As talked with Oussama, I have only two explanations for `CallType`:
```
CALL_TYPE_NORMAL => Only Audio
CALL_TYPE_VIDEO => Video/Audio
CALL_TYPE_FORCED_AUDIO = No explanation yet.
```

Those will be added in `Calling.kt` on top of `object CallType` as a doc.

Reply:
`CALL_TYPE_FORCED_AUDIO` was used for legacy groups calls (before conference calls). If the number of participants were above a certain limit you weren't allowed join with video.

Reply:
So I think these kind of "deprecated" things can have their definitions inside platform specific-code if necessary.

But I think we can try our best to avoid making them visible to consumers of `logic/commonMain`.
That is: If it isn't supported, they _should_ not be visible.

If I understood correctly, this code here follows this principle, as `CallType` is defined in `:calling`.